code/game/turfs/open/chasm.dm

```diff
@@ -79,6 +79,8 @@
 /// Handles adding the chasm component to the turf (So stuff falls into it!)
 /turf/open/chasm/proc/apply_components(mapload)
 	AddComponent(/datum/component/chasm, GET_TURF_BELOW(src), mapload)
+	if(istype(loc, /area/deathmatch)) // dynamite fishing was not very thought out
+		qdel(GetComponent(/datum/component/fishing_spot)) // KILL 6 BILLION DEM- LOBSTROSITIES
```

Comment:
can this be made to just not apply it in the first place? chasm initialize maybe?